src/test/java/io/aiven/kafka/connect/gcs/config/GcsSinkConfigTest.java

```diff
@@ -715,4 +716,45 @@ void correctFilenameTimestampSource() {
 
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {"jsonl", "csv"})
+    void supportedFormatTypeConfig(final String formatType) {
+        final Map<String, String> properties = new HashMap<>();
+        properties.put("gcs.bucket.name", "test-bucket");
+        properties.put(GcsSinkConfig.FORMAT_OUTPUT_TYPE_CONFIG, formatType);
+
+
+        final GcsSinkConfig c = new GcsSinkConfig(properties);
+        final FormatType expectedFormatType = FormatType.forName(formatType);
+
+        assertEquals(expectedFormatType, c.getFormatType());
+    }
+
+    @Test
+    void emptyFormatType() {
```

Comment:
Any reason not to cover this in `correctMinimalConfig`?

Reply:
No. Just Forgot one.
- [ ] Add a use case to a `correctMinimalConfig`